### ISSUE TITLE
Req: Support opening TIF, PNG, and JPG images

### ISSUE THREAD

**Issue by xiota** [CarVac/filmulator-gui#120]
It would be nice if TIF, PNG, and JPG images could be opened.

* The algorithm appears to operate on data after demosaicing. If so, raw information isn't strictly necessary.
* There are a number of open issues related to DNG files. The ability to work on 16-bit TIF files would serve as a reasonable workaround until those issue could be resolved directly.
* The old command-line filmulator was able to process TIF and JPG images.

**Comment by abnormally-distributed** [comment]
I agree, it would be nice because the program doesn't like raw files from certain cameras
